lib/cog/commands/helpers.ex

```diff
@@ -133,6 +133,8 @@ defmodule Cog.Commands.Helpers do
     do: "Could not find '#{resource_type}' with the name '#{resource_name}'"
   def error(:invalid_handle),
     do: "Invalid chat handle"
+  def error({:trigger_invalid, %Ecto.Changeset{}=changeset}),
+    do: changeset_errors(changeset)
```

Comment:
I was pretty much just copying the bits that were already there. Though it may not be a terrible idea to clean some of these helpers up a bit. Or should we just create an issue for later?